client/components/tinymce/plugins/after-the-deadline/core.js

```diff
@@ -0,0 +1,610 @@
+/*
+ * atd.core.js - A building block to create a front-end for AtD
+ * Author      : Raphael Mudge & Andrew Duthie, Automattic
+ * License     : LGPL
+ * Project     : http://www.afterthedeadline.com/developers.slp
```

Comment:
This URL is invalid - I think it should be http://www.afterthedeadline.com/development.slp